x/oracle/keeper/keeper.go

```diff
@@ -35,3 +38,40 @@ func NewKeeper(
 		paramSpace: paramSpace,
 	}
 }
+
+func (k Keeper) VerifySignature(ctx sdk.Context, msg codec.ProtoMarshaler, sigBz []byte) error {
```

Comment:
Why don't we name this function more specific? e.g. `VerifyOracleSignature`.
It seems too general to me.

Reply:
fixed: https://github.com/medibloc/panacea-core/pull/533/commits/fec02e4707f772c7ada78e92414e0aa8d2ea5a30
Thank you.